conversion.php

```diff
@@ -1,33 +1,75 @@
 <?php
 
 function fahrenheit_to_celsius ($temp) {
+    if (gettype($temp == 'int')) {

```

Comment:
I think we can ignore this if statement because we already set number input.
 Value: <input type="number" name="Val" >
But it will be ok to keep it. It's just my comment.

Reply:
Good point - I didn't think about that.